package-lock.json

```diff
@@ -0,0 +1,9722 @@
+{
```

Comment:
This file should be removed. We use yarn.

Reply:
removed